docs/usage/reading-list.md

```diff
@@ -0,0 +1,74 @@
+# Reading list
+
+Renovate's documentation has a lot of pages.
+To ease you into using Renovate we created reading lists.
+The reading lists contain the essential information for each type of user.
+
+How much you should read depends on how much you want to customize Renovate's behavior.
+
+## How to use this page
+
+We created reading lists for these types of users:
+
+- Beginners
+- Intermediate
+- Advanced
+
+Start with the "Beginners" reading list.
+If you're self-hosting or need to update private packages, complete the relevant reading lists for those.
+
+## I don't know where to start
+
+If you're new to Renovate, you should:
+
+- Use the Mend-hosted GitHub App, or let somebody else self-host Renovate for you
+- Stick with the `config:base` or `config:best-practices` preset
+- Use the Dependency Dashboard (`config:base` or `config:best-practices` enables it automatically)
+- Read the pages in the "Beginners" list
+- Only create custom Renovate configuration when really needed
+
+## Beginners
+
+Start by reading:
+
+- [Installing & Onboarding](./getting-started/installing-onboarding.md)
+- [Key concepts, Dependency Dashboard](./key-concepts/dashboard.md)
+- [Use Cases](./getting-started/use-cases.md)
+- [Running Renovate](./getting-started/running.md)
+- [Troubleshooting](./troubleshooting.md)
+- [Known limitations](./known-limitations.md)
+- [Release notes for major versions](./release-notes-for-major-versions.md)
```

Comment:
Are you happy keeping it to just a list, or should I have some introductory text? Here's how the Beginners section could look:

> First, you'll have to install Renovate, read Installing & Onboarding to learn how.
> Once you've got Renovate up-and-running, check out your Dependency Dashboard, it shows all your pending updates.
> Then read about the Use Cases that people have for using Renovate.
>
> If you're having a problem with Renovate, read the Troubleshooting guide.
>
> Do you think Renovate is not responding, or merges updates slowly? First read the Known Limitations to learn what you can reasonably expect.
>
> And finally, we have Release notes for major versions of Renovate. These come with developer commentary, and tell you _why_ we're making certain changes.
>
> Whew, that was a lot of reading, but you're up to speed now.
> We hope you enjoy Renovate, and your updated dependencies!

Let me know what you want! 😉

Reply:
I'll let @rarkins decide if they want a short list of things to read, or longer more descriptive paragraphs for each item. A longer text:

- risks duplicating content with other pages
- will probably be skimmed over by most readers anyway
- makes the text friendlier
- gives context for _why_ we want users to read certain pages

Reply:
Short is fine